addons/microsoft_calendar/models/microsoft_sync.py

```diff
@@ -56,7 +56,8 @@ class MicrosoftSync(models.AbstractModel):
     _description = "Synchronize a record with Microsoft Calendar"
 
     microsoft_id = fields.Char('Microsoft Calendar Id', copy=False)
-    need_sync_m = fields.Boolean(default=True, copy=False)
+    need_sync_m = fields.Boolean(default=True, copy=False,
+        help="This field helps to known when a microsoft event need to be resynced")
```

Comment:
No need to add a translation, you can put a comment above the field definition. :)